src/services/AudioMixer.cxx

```diff
@@ -367,11 +366,11 @@ void AudioMixer::playSoundtrackWithEffect(SoundtrackUPtr& track, int ALEffect)
   alDeleteEffects(1, &effect);
 
   //apply effect to source
-  alSource3i(track->source, AL_AUXILIARY_SEND_FILTER, (ALint)(track->effect_slot), 0, AL_FILTER_NULL);
-  assert(alGetError() == AL_NO_ERROR && "Failed to setup effect for sound source send 0.");
+  alSource3i(track->source[track->isMusic], AL_AUXILIARY_SEND_FILTER, (ALint)(track->effect_slot), 0, AL_FILTER_NULL);
```

Comment:
isMusic should be changed here
it's not readable, as I suggested before maybe adding enum will make things easier to read

Reply:
Oh, I thought it was pretty readable/understandable what it was, but I guess you're right, an enumerator might help with readability. I'll try it out and see how it looks and feels.

Reply:
So what would the ideal line be, 
`alSource3i(track->source[MUSIC]...` and `alSource3i(track->source[SFX]...`?

Reply:
make more sense